main/main.c

```diff
@@ -775,6 +797,7 @@ PHP_INI_BEGIN()
 #endif
 	STD_PHP_INI_ENTRY("syslog.facility",		"LOG_USER",		PHP_INI_SYSTEM,		OnSetFacility,		syslog_facility,	php_core_globals,		core_globals)
 	STD_PHP_INI_ENTRY("syslog.ident",		"php",			PHP_INI_SYSTEM,		OnUpdateString,		syslog_ident,		php_core_globals,		core_globals)
+	STD_PHP_INI_ENTRY("syslog.filter",		"ascii",		PHP_INI_ALL,		OnSetLogFilter,		syslog_filter,		php_core_globals, 		core_globals)
```

Comment:
This is not acceptable default as it could break all application using UTF-8 - it needs to be either `none` or `no-ctrl`. I think that changing it to `no-ctrl` might be fine as users shouldn't probably use control characters by default but not sure about the horizontal tab. Might be safer to leave it as `none`...